docs/standard/microservices-architecture/secure-net-microservices-web-applications/authorization-net-microservices-web-applications.md

```diff
@@ -27,11 +27,11 @@ public class AccountController : Controller
 
 By default, adding an Authorize attribute without parameters will limit access to authenticated users for that controller or action. To further restrict an API to be available for only specific users, the attribute can be expanded to specify required roles or policies that users must satisfy.
 
-## Implementing role-based authorization
+## Implement role-based authorization
 
-ASP.NET Core Identity has a built-in concept of roles. In addition to users, ASP.NET Core Identity stores information about different roles used by the application and keeps track of which users are assigned to which roles. These assignments can be changed programmatically with the RoleManager type (which updates roles in persisted storage) and UserManager type (which can assign or unassign users from roles).
+ASP.NET Core Identity has a built-in concept of roles. In addition to users, ASP.NET Core Identity stores information about different roles used by the application and keeps track of which users are assigned to which roles. These assignments can be changed programmatically with the RoleManager type (which updates roles in persisted storage) and UserManager type (which can assign or unassigned users from roles).
```

Comment:
I think "unassigned" was correct.

Reply:
You're right, but spell checker marked it as a typo, so I rephrased it to "grant or revoke roles from users", that seems better (IMHO).